docs/docs/guides/otel.md

```diff
@@ -160,3 +160,34 @@ kubectl port-forward deployment/metrics-operator 9999 -n keptn-system
 ```
 
 You can access the metrics from your browser at: `http://localhost:9999`
+
+# Advanced tracing configurations in Keptn: Linking traces
```

Comment:
you can only have one h1 at the top of the file so this will need to be an h2 or h3